WatchFaceFormat/validator-plugin/plugins/src/main/kotlin/ValidatorExec.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.gradle.api.GradleException
+import org.gradle.api.Project
+import java.nio.file.Path
+
+
+/**
+ * Runs the validator against WFF XML files found in the project structure, matching any files:
+ *
+ *    res\raw*\*.xml
```

Comment:
```suggestion
 *    res/raw*/*.xml
```

Reply:
Yeah, I did this because the `/*` in the path mucked up the comment, so it was invalid syntax. I'll try something else.

Reply:
This works: `res/raw[-qualifier]/*.xml` ?